Makefile

```diff
@@ -83,7 +83,7 @@ test: test-unit ## Run the tests
 
 TEST_PKGS:=$(shell go list ./...)
 test-unit: ## Run the unit tests
-	$(Q)go test -count=1 -short ${TEST_PKGS}
+	$(Q)go test -count=1 -short ${TEST_PKGS} -v
```

Comment:
ihmo only revert this one, rebase and all shows fine for me. 
Really nice contribution.